### ISSUE TITLE
Not all types from chrome present in the package

### ISSUE THREAD

**Issue by NN---** [Lusito/webextension-polyfill-ts#11]
I tried to replace each chrome with browser and this code didn't compile:

```js
import TabActiveInfo = browser.tabs.TabActiveInfo;
```

**Comment by Lusito** [comment]
Not sure which type definitions you had before, but there is no type TabActiveInfo in web-extensions. Neither in the MDN documentation nor in the schema.json files from Mozilla.

DefinitelyTyped seems to have it, so I'm guessing you've been using this. A lot of these types are not defined in the schema files, so they are autogenerated in webextension-polyfill-ts. DefinitelyTyped wrote this by hand, so they could fine-tune it.

In this case, I'm assuming you are interested in the type for the onActivated parameter. For webextension-polyfill-ts, this type is automatically as [OnActivatedActiveInfoType](https://github.com/Lusito/webextension-polyfill-ts/blob/master/src/generated/tabs.ts#L806) (see [onActivated](https://github.com/Lusito/webextension-polyfill-ts/blob/master/src/generated/tabs.ts#L1201).

I know, not as nicely named as TabActiveInfo. It's not as easy to avoid when auto-generating names, which are not present in the schemas (and there are a lot of them). It would be nice to have a map of autogenerated names

Long story short: You can import these types like this:
```typescript
import { Tabs } from "webextension-polyfill-ts";

function onTabActivated(activeInfo: Tabs.OnActivatedActiveInfoType) {
    // ...
}
```

Or, if you want to keep using the identifier TabActiveInfo:
```typescript
import { Tabs } from "webextension-polyfill-ts";
type TabActiveInfo = Tabs.OnActivatedActiveInfoType;

function onTabActivated(activeInfo: TabActiveInfo ) {
    // ...
}
```

Hope this helps :)

**Comment by NN---** [comment]
Everything is correct.
Too bad the switch from DefinitelyType is not as smooth as expected.

I guess there is nothing to do but to maintain the mapping.
Many people would use this project if the transition was smooth.
